examples/pojo-jar/src/main/java/org/apache/camel/OpenApiRoute.java

```diff
@@ -0,0 +1,53 @@
+// camel-k: name=pojo-jar
+// camel-k: dependency=camel-jackson
+// camel-k: dependency=camel-openapi-java
+// camel-k: dependency=mvn:org.projectlombok:lombok:1.18.22
```

Comment:
Would this be possible to remove Lombok to keep the example un-opinionated?

Reply:
Yes sure. @davsclaus what do you think as you explicitly asked for it in the issue comment.